kot/common/crawl.py

```diff
@@ -1,5 +1,6 @@
 import random
 import time
+import os
```

Comment:
Please sort imports alphabetically
```suggestion
import os
import random
import time
```